src/SFA.DAS.Campaign.Application/Content/Queries/GetArticleQueryHandler.cs

```diff
@@ -22,10 +22,11 @@ public GetArticleQueryHandler(IApiClient apiClient, IOptions<CampaignConfigurati
 
         public async Task<GetArticleQueryResult<Article>> Handle(GetArticleQuery request, CancellationToken cancellationToken)
         {
-            var canPreview = _config.Value.AllowPreview;
+            var allowPreview = _config.Value.AllowPreview;
+            var forcePreview = _config.Value.ForcePreview;
             Page<Article> article = null;
 
-            if (canPreview && request.Preview)
+            if (allowPreview && request.Preview || forcePreview)
```

Comment:
When using && with || in the same if statement it is good practice to parenthesis your intention e.g. above is equivalent to **if((allowPreview && request.Preview) || forcePreview)** - so this can forces a preview when it's not allowed and not requested - was that your intention?